pkg/codegen/go/utilities.go

```diff
@@ -0,0 +1,20 @@
+package gen
+
+import "github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
+
+// TODO: either fill out rest of providers or find more scalable solution
```

Comment:
We should pull this from the package's schema (see `hcl2.Program.Packages()`). In the fullness of time, we might need to link from a schema type back to its package in the event that different versions of the same package are in use).